README.md

```diff
@@ -38,132 +38,140 @@ Then we need to include carbon-components in `src/styles.scss`:
 @use "@carbon/styles";
 ```
 
-> *Note:* For offline usage we'll need to set `$font-path: '~carbon-components/src/globals/fonts';` at the very top of our `src/styles.scss`. This will copy the fonts to our `dist` folder upon successful build. If you like the fonts to be a part of your `assets` folder and not pollute the `dist` folder then copy the fonts from `node_modules/carbon-components/src/globals/fonts` into our app's `src/assets/fonts` folder and add `$font-path: '/assets/fonts/';` at the very top of our `src/styles.scss`.
+> _Note:_ For offline usage we'll need to set `$font-path: '~carbon-components/src/globals/fonts';` at the very top of our `src/styles.scss`. This will copy the fonts to our `dist` folder upon successful build. If you like the fonts to be a part of your `assets` folder and not pollute the `dist` folder then copy the fonts from `node_modules/carbon-components/src/globals/fonts` into our app's `src/assets/fonts` folder and add `$font-path: '/assets/fonts/';` at the very top of our `src/styles.scss`.
```

Comment:
@IgnacioBecerra What exactly is updated in readme? Based on what I see, there really isn't any changes apart from spacing 🤔

Reply:
@Akshat55 Ah seems like VSCode applied some unneeded formatting when saving the file. The wording of the Telemetry notice under the README is what is different:
https://github.com/carbon-design-system/carbon-components-angular/pull/3027/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L171-L174

Reply:
Thanks for the clarification!